index.js

```diff
@@ -68,6 +69,9 @@ module.exports = function({utPort, registerErrors, vfs, joi}) {
             return {
                 type: 'object',
                 properties: {
+                    retryOnDeadlock: {
+                        type: ['boolean']
```

Comment:
why array?

Reply:
because it is a typo. First I made it ['boolean', 'integer'] so that in case of an integer then to be assumed it indicates the retry count. But then I realized it should not be necessary to retry more than once. Fixing it...